data_summaries/Lumos Planning.json

```diff
@@ -0,0 +1,168 @@
+{
+    "lumos-planning-complex_qa": {
+        "Unique Dataset Identifier": "lumos-planning-complex_qa",
+        "Dataset Name": "ai2lumos/lumos_unified_plan_iterative",
+        "Paper Title": "Lumos: Learning Agents with Unified Data, Modular Design, and Open-Source LLMs",
+        "Dataset URL": "https://huggingface.co/datasets/ai2lumos/lumos_unified_plan_iterative",
+        "GitHub URL": "https://github.com/allenai/lumos",
+        "Hugging Face URL": "https://huggingface.co/datasets/ai2lumos/lumos_unified_plan_iterative",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2311.05657",
+        "Semantic Scholar Corpus ID": 265128672,
+        "Collection": "Lumos Planning",
+        "Collection URL": "https://huggingface.co/datasets/ai2lumos/lumos_unified_plan_iterative",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Logical Reasoning Question Answering",
+            "Binary Question Answering",
+            "Open-Domain Question Answering",
+            "Dialogue Generation",
+            "Instruction Following"
+        ],
+        "Text Sources": [
+            "ML Datasets"
```

Comment:
Each place we put "ML Datasets" can we also add the sources of text that those datasets came from. I know this is a bit more work, but very important for us to clearly document these sources -- its key to our later ecosystem analysis. Thanks!